packages/team-management/src/components/TeamsAddEdit/index.tsx

```diff
@@ -68,7 +68,7 @@ function setupInitialValue(
     .then((response) => {
       setInitialValue({
         ...response,
-        practitioners: response.practitioners.map((prac) => prac.identifier),
+        practitioners: response.practitioners.map((practitioner) => practitioner.name),
```

Comment:
@machariamuguku I think this should atleast require a test to get updated or a new test added. of interest, I would like to know what implications this would mean if more than one practitioner share a name

Reply:
@p-netm This is only used for the select display. If practitioners share a name both will be shown, similar names and all

Reply:
cool, please add a test

Reply:
if they will show similar name then wouldn't there be duplicate keys or related error ?

Reply:
@ashfahan only names, the keys and value are unique (identifier)